system_tests/common_test.go

```diff
@@ -60,6 +59,172 @@ import (
 type info = *BlockchainTestInfo
 type client = arbutil.L1Interface
 
+type SecondNodeParams struct {
+	nodeConfig           *arbnode.Config
+	execConfig           *gethexec.Config
+	stackConfig          *node.Config
+	dasConfig            *das.DataAvailabilityConfig
+	initData             *statetransfer.ArbosInitializationInfo
+	useBuilderNodeConfig bool
```

Comment:
Doesn't it make more sense for a test to just do:
```
 SecondNodeParams.nodeConfig = builder.nodeConfig
 ```
 instead of using this bool?

Reply:
Agreed, since `arbnode.ConfigDefaultL1NonSequencerTest()` is more widely used then the previous built l2 node's config in the current tests